api/src/main/scala/uk/ac/wellcome/platform/api/controllers/ContextController.scala

```diff
@@ -0,0 +1,21 @@
+package uk.ac.wellcome.platform.api.controllers
+
+import com.twitter.inject.annotations.Flag
+import com.twitter.finagle.http.Request
+import com.twitter.finatra.http.Controller
+
+import javax.inject.{Inject, Singleton}
+
+
+@Singleton
+class ContextController @Inject()(
+  @Flag("api.prefix") apiPrefix: String) extends Controller {
```

Comment:
Will change the line layout, I'm style getting my head around Scala style...

@Flag is the Finatra way to do this, I introduced it as part of the Swagger support. So I'd vote to keep for now, then revisit when we come back to the API properly and decide if we want to use @Flag or not across the board.